backup_preload_mongo.sh

```diff
@@ -0,0 +1,1673 @@
+  dnsNames:
+    - mongodb.root
+  duration: 17520h
+  isCA: true
+  issuerRef:
+    kind: Issuer
+    name: god-issuer
+  secretName: mongodb-root-ca-cert
+EOF
+    #mongodb-root-ca-issuer-issuer.yaml
+    cat << EOF | oc apply -f -
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: mongodb-root-ca-issuer
+  labels:
+    app.kubernetes.io/instance: mongodbs.operator.ibm.com
+    app.kubernetes.io/managed-by: mongodbs.operator.ibm.com
+    app.kubernetes.io/name: mongodbs.operator.ibm.com
+spec:
+  ca:
+    secretName: mongodb-root-ca-cert
+EOF
+    #namespace-scope-cm.yaml
+    cat << EOF | oc apply -f -
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: namespace-scope
+data:
+  namespaces: "$TO_NAMESPACE"
+EOF
+    #apply statefulset (in same dir)
+    #icp-mongodb-ss.yaml
+    cat << EOF | oc apply -f -
+kind: StatefulSet
+apiVersion: apps/v1
+metadata:
+  name: icp-mongodb
+  labels:
+    app: icp-mongodb
+    app.kubernetes.io/instance: mongodbs.operator.ibm.com
+    app.kubernetes.io/managed-by: mongodbs.operator.ibm.com
+    app.kubernetes.io/name: mongodbs.operator.ibm.com
+    release: mongodb
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: icp-mongodb
+      release: mongodb
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        app: icp-mongodb
+        app.kubernetes.io/instance: common-mongodb
+        release: mongodb
+      annotations:
+        clusterhealth.ibm.com/dependencies: ibm-common-services.cert-manager
+        productID: 068a62892a1e4db39641342e592daa25
+        productMetric: FREE
+        productName: IBM Cloud Platform Common Services
+        prometheus.io/path: /metrics
+        prometheus.io/port: '9216'
+        prometheus.io/scrape: 'true'
+    spec:
+      restartPolicy: Always
+      initContainers:
+        - resources:
+            limits:
+              cpu: '1'
+              memory: 640Mi
+            requests:
+              cpu: 500m
+              memory: 640Mi
+          terminationMessagePath: /dev/termination-log
+          name: install
+          command:
+            - /install/install.sh
+          imagePullPolicy: IfNotPresent
+          volumeMounts:
+            - name: mongodbdir
+              mountPath: /work-dir
+              subPath: workdir
+            - name: configdir
+              mountPath: /data/configdb
+            - name: config
+              mountPath: /configdb-readonly
+            - name: install
+              mountPath: /install
+            - name: keydir
+              mountPath: /keydir-readonly
+            - name: ca
+              mountPath: /ca-readonly
+            - name: mongodbdir
+              mountPath: /data/db
+              subPath: datadir
+            - name: tmp-mongodb
+              mountPath: /tmp
+          terminationMessagePolicy: File
+          image: >-
+            icr.io/cpopen/cpfs/ibm-mongodb-install@sha256:bb236428cd36f3937d268c4475a1c62ac1a4e7cb9ca0f3de482f08817378b003
```

Comment:
It should be the same image as in the CP2 namespace - otherwise it might not be mirrored